Commands/Branding/SetProvisioningTemplateMetadata.cs

```diff
@@ -16,33 +16,27 @@ namespace SharePointPnP.PowerShell.Commands.Branding
         Category = CmdletHelpCategory.Branding)]
     [CmdletExample(
      Code = @"PS:> Set-SPOProvisioningTemplateMetadata -Path template.xml -TemplateDisplayName ""DisplayNameValue""",
-     Remarks = @"Sets the DisplayName property of a provisioning template in XML format.
-",
+     Remarks = @"Sets the DisplayName property of a provisioning template in XML format.",
      SortOrder = 1)]
     [CmdletExample(
      Code = @"PS:> Set-SPOProvisioningTemplateMetadata -Path template.pnp -TemplateDisplayName ""DisplayNameValue""",
-     Remarks = @"Sets the DisplayName property of a provisioning template in Office Open XML format.
-",
+     Remarks = @"Sets the DisplayName property of a provisioning template in Office Open XML format.",
      SortOrder = 2)]
     [CmdletExample(
      Code = @"PS:> Set-SPOProvisioningTemplateMetadata -Path template.xml -TemplateImagePreviewUrl ""Full URL of the Image Preview""",
-     Remarks = @"Sets the DisplayName property of a provisioning template in XML format.
-",
+     Remarks = @"Sets the DisplayName property of a provisioning template in XML format.",
```

Comment:
Incorrect help message. It sets the the Url to the preview image in an XML template.